imports/plugins/included/product-variant/containers/productGridItemsContainer.js

```diff
@@ -45,7 +45,7 @@ const wrapComponent = (Comp) => (
       document.querySelector(".page > main").removeEventListener("click", this.onPageClick);
     }
 
-    onPageClick = () => {
+    onPageClick = (event) => {
```

Comment:
is there a global `event` variable in Chrome? surprised this worked anywhere!

Reply:
The `event` object is always available when an event takes place in the browser.